Justpaste.it.js

```diff
@@ -0,0 +1,125 @@
+{
+	"translatorID": "1d1a51d4-60bf-44b8-ae28-8b154d1ed721",
+	"label": "Justpaste.it",
```

Comment:
This is the same ID as you used in #1131, but the same ID cannot be used for two different translators.